Adaptors/RabbitMQ/src/PullQueueStorage.cs

```diff
@@ -69,11 +69,12 @@ await ConnectionRabbit.Init(cancellationToken)
                       },
                     };
 
-    ConnectionRabbit.Channel!.QueueDeclare(Options!.PartitionId,
-                                           false, /* to survive broker restart */
-                                           false, /* used by multiple connections */
-                                           false, /* not deleted when last consumer unsubscribes (if it has had one) */
-                                           queueArgs);
+    (await ConnectionRabbit.GetConnectionAsync(cancellationToken)
+                           .ConfigureAwait(false)).QueueDeclare(Options!.PartitionId,
```

Comment:
I would split this into 2 statements: one for getting the connection, and the other declaring the queue.